### ISSUE TITLE
Roadblocks in converting from Hugo Academic

### ISSUE THREAD

**Issue by spcanelon** [hugo-apero/apero#78]
Hi @apreshill!

Thanks for [outlining some steps on Twitter](https://twitter.com/apreshill/status/1370027805543698432?s=20) on how to approach converting to Apéro from Hugo Academic. I followed the steps (listed below) but am unable to render the site using `blogdown::serve_site()` 😢 

Steps followed, with corresponding commit in my own repo:
1. Install theme alongside Academic (https://github.com/spcanelon/silvia/commit/cc5d24d93676990675abc52145fd7a369c7bffa6), change in `config.toml`(https://github.com/spcanelon/silvia/commit/499cc959a947761b4dbc518f5a1bf6312527b517)
2. Copy all Academic shortcodes to `layouts/` root (https://github.com/spcanelon/silvia/commit/f3c7d5334b4effbd850b204eb267425f6740b4af)
3. Remove all assets (https://github.com/spcanelon/silvia/commit/3843c76a5da6184b2d9b547b18f96ec6810a695a)
4. Remove all custom layouts (https://github.com/spcanelon/silvia/commit/1ad7e3d491d309e71e1b4fa0bbad1e3af6b9d322)
5. Copy over Apéro example site config.toml file (https://github.com/spcanelon/silvia/commit/db37289e768640522130f98353996de4a6e0abfc)
6. Remove Academic config directory (https://github.com/spcanelon/silvia/commit/5541f38871911d5067c6c8856936d54d183b3ec9)

The error I'm stuck on at the moment is the following:

> Error: Error building site: failed to render pages: render of "page" failed: execute of template failed: template: _default/single.html:3:8: executing "_default/single.html" at <partial "head.html" .>: error calling partial: "/Users/scanelon/Documents/Repos/Website/silvia/themes/hugo-apero/layouts/partials/head.html:14:53": execute of template failed: template: partials/head.html:14:53: executing "partials/head.html" at <js>: can't evaluate field Build in type string

I'm understanding this to mean there's an issue with using the templates in `themes/hugo-apero`, but I'm not sure why. I'm sure I missed something along the way 😅 
Any pointers you have would be much appreciated!

**Comment by johngoldin** [comment]
I tried to use the RStudio new blowdown site feature or calling new_site() with hugo-apero/apero as the theme and got exactly the same error. I have an existing blog that is using beautifulhugo, but that is in a different project. I get exactly the same error as described here, except it refers to my new apes project folder. So this does not relate specifically to converting from Academic.

**Comment by johngoldin** [comment]
Progress: rather than go the blowdown::new_site() route, I cloned https://github.com/hugo-apero/apero onto my Mac. After doing blogdown::install_hugo("0.79.0") (from version 0.59) I was successful at getting the site to start up via serve_site(). At this point I'm not sure whether the key was simply to upgrade to 0.79 or whether I also needed to clone the repo rather than use blowdown::new_site(), but in any case I can move on from here to see where I fail next. Also I supposed I better go back to my production web site to see whether I have broken it after the Hugo upgrade.

**Comment by spcanelon** [comment]
Thanks for sharing @johngoldin! I am currently on Hugo v0.69.2 so your success with updating to v0.79 gives me hope. I'll try that next and report back.

**Comment by spcanelon** [comment]
Thanks for the guidance @johngoldin and @apreshill, updating the Hugo version worked! Here are the steps I took:

1. `blogdown::install_hugo()` -- installed latest version of Hugo (v0.82.1)
2. Updated `.Rprofile` per instructions, and restarted R
3. Updated `netlify.toml`
4. Updated Hugo version on Netlify.com
5. `blogdown::check_site()` -- suggested I use `blogdown::install_hugo("0.82.1")` so I did, but I'm not sure why I needed to do it again. I made sure to restart R here also.
6. `blogdown::check_site()` -- all checks passed
7. `blogdown::serve_site()` -- site preview worked! 🎉
